dashboard/global.R

```diff
@@ -0,0 +1,110 @@
+# In the line below, import the shiny library so that it's available
+# in both ui.R and server.R
+library(readxl)
+library(shiny)
+library(tidyverse)
+library(DT)
+library(here)
+
+library(odbc)       #R library for Open Database Connectivity, used to connect to databases
+library(RODBC)      #Manage DB connections
+
+
+
+
+
+
+here()
+
+# In the lines below, import the files and process them.
+
+SMR_accuracy <- read_csv(here("./smr_data/SMR_accuracy.csv"))
```

Comment:
The function `here()` should have each directory and filename as its own argument. You've done this when you've used it elsewhere so assume it's just a typo 😄 

```suggestion
SMR_accuracy <- read_csv(here("smr_data", "SMR_accuracy.csv"))
```